spec/01-lexical-syntax.md

```diff
@@ -553,6 +553,22 @@ but are required to be properly nested.  Therefore, a comment like
 `/* /* */` will be rejected as having an unterminated
 comment.
 
+## Multi-line and trailing commas
+
+Trailing commas are allowed when followed by a newline and a closing
```

Comment:
By the way, when looking at the implementation again, I noticed that the only other use of `afterLineEnd` is predicated on `!applyBracePatch` (presumably for the IDE). Should we do that here too?